proto/viam/service/motion/v1/motion.proto

```diff
@@ -35,6 +37,33 @@ service MotionService {
     };
   }
 
+  rpc MoveOnGlobeNew(MoveOnGlobeNewRequest) returns (MoveOnGlobeNewResponse) {
+    option (google.api.http) = {
+      get: "/viam/api/v1/service/motion/{name}/move_on_globe"
```

Comment:
new

Reply:
does this not need a separate path?

Reply:
it does, this was a note to myself I need to add it